frontend/src/metabase/query_builder/components/QueryVisualization.jsx

```diff
@@ -104,7 +104,11 @@ export const VisualizationEmptyState = ({ className }) => (
 export function VisualizationRunningState({ className = "" }) {
   const [isSlow] = useTimeout(SLOW_MESSAGE_TIMEOUT);
 
-  const getLoadingMessage = useSelector(getWhiteLabeledLoadingMessageFactory);
+  const getLoadingMessage = useSelector(getWhiteLabeledLoadingMessageFactory, {
```

Comment:
:x: This warning is correct and can be fixed by making sure [`getLoadingMessageFactory` returns the same function](https://github.com/metabase/metabase/blob/d6d2edb3e54530c0d871d0a2a7060361a2c80f80/frontend/src/metabase/plugins/index.ts#L213) every time.